migrations/20170222000549_builds_number.js

```diff
@@ -0,0 +1,22 @@
+exports.up = async (knex) => {
+  await knex.schema
+    .table('builds', (table) => {
+      table.integer('number').index()
+    })
+  await knex.raw(`
+    UPDATE builds SET number = numbers.number
+    FROM (
+      SELECT id, row_number() OVER (PARTITION BY "repositoryId" ORDER BY id) AS number
```

Comment:
Looks like it's working 👍 
![capture d ecran 2017-02-23 a 09 29 11](https://cloud.githubusercontent.com/assets/3165635/23250699/9287a5f4-f9aa-11e6-8bf3-b3a1bb451fa0.png)